back/qc_api/util/decorator.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework.response import Response
+from rest_framework import status
+
+
+def catch_bad_request(func):
```

Comment:
I think I've already merged the function which does the same job as this in ```qc_api/views/utils.```
Please check this and use the one which you prefer, since I don't mind using the either one. Although, I think there's a need to make a consensus on how to store the utility functions.

Reply:
#53